book/variables_and_subexpressions.md

```diff
@@ -63,3 +63,21 @@ We can do a very similar action in a single step using a subexpression path:
 ```
 
 It depends on the needs of the code and your particular style which form works best for you.
+
+## Short-hand subexpressions
```

Comment:
These are called "row conditions" in the engine code. Not sure if that's clearer or less clear than "short-hand subexpressions"

Reply:
Changed the header to `## Short-hand subexpressions (row conditions)`. Think that strikes a balance between:
1) Fitting the context of the Variables and Subexpressions page
2) Tying things back to the engine's terminology